__tests__/app.test.js

```diff
@@ -126,3 +132,26 @@ describe("GET /api/articles/:article_id/comments", () => {
     }
   });
 });
+describe("Post /api/articles/:article_id/comments", () => {
+  test("posts a comment to an article", async () => {
+    const response = await request(app).post("/api/articles/1/comments").send({
+      username: "butter_bridge",
+      body: "This is a test comment",
+    });
+    expect(response.status).toBe(201);
+    const { comment } = response.body;
+    expect(comment).toHaveProperty("comment_id");
+    expect(comment).toHaveProperty("author", "butter_bridge");
+    expect(comment).toHaveProperty("body", "This is a test comment");
+    expect(comment).toHaveProperty("article_id", 1);
+  });
+
+  test("responses with a 400 error if no article details are provided", async () => {
+    const commentData = {};
```

Comment:
we are missing some error tests 
- what if path is /articles/1000
- what if path is articles/hello
- what if username doesnt exist